code/modules/clothing/spacesuits/void.dm

```diff
@@ -1,14 +1,131 @@
+//Voidsuits
+/obj/item/clothing/head/helmet/space/voidsuit
 	name = "NASA Void Helmet"
-	desc = "A high tech, NASA Centcom branch designed, dark red space suit helmet. Used for AI satellite maintenance."
-	icon_state = "void"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcomm. This one is in red."
+	icon_state = "void-red"
 	item_state = "void"
+	flags_inv = HIDEMASK|HIDEEARS
 
-/obj/item/clothing/suit/space/nasavoid
-	name = "NASA Voidsuit"
-	icon_state = "void"
+/obj/item/clothing/suit/space/voidsuit
+	name = "NASA Void Suit"
+	icon_state = "void-red"
 	item_state = "void"
-	desc = "A high tech, NASA Centcom branch designed, dark red Space suit. Used for AI satellite maintenance."
-	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/multitool)
\ No newline at end of file
+	desc = "A retro Space suit designed by a branch at NASA Centcomm. This one is in red."
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/multitool)
+
+//Colors!!!
+/obj/item/clothing/head/helmet/space/voidsuit/green
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcomm. This one is in green."
+	icon_state = "void-green"
+
+/obj/item/clothing/suit/space/voidsuit/green
+	name = "NASA Void Suit"
+	icon_state = "void-green"
+	desc = "A retro Space suit designed by a branch at NASA Centcomm. This one is in green."
+
+/obj/item/clothing/head/helmet/space/voidsuit/ntblue
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcomm. This one is in NT colors."
+	icon_state = "void-ntblue"
+
+/obj/item/clothing/suit/space/voidsuit/ntblue
+	name = "NASA Void Suit"
+	icon_state = "void-ntblue"
+	desc = "A retro Space suit designed by a branch at NASA Centcomm. This one is in NT colors."
+
+/obj/item/clothing/head/helmet/space/voidsuit/purple
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcomm. This one is in purple."
+	icon_state = "void-purple"
+
+/obj/item/clothing/suit/space/voidsuit/purple
+	name = "NASA Void Suit"
+	icon_state = "void-purple"
+	desc = "A retro Space suit designed by a branch at NASA Centcomm. This one is in purple."
+
+/obj/item/clothing/head/helmet/space/voidsuit/yellow
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcomm. This one is in yellow."
+	icon_state = "void-yellow"
+
+/obj/item/clothing/suit/space/voidsuit/yellow
+	name = "NASA Void Suit"
+	icon_state = "void-yellow"
+	desc = "A retro Space suit designed by a branch at NASA Centcomm. This one is in yellow."
+
+/obj/item/clothing/head/helmet/space/voidsuit/ltblue
+	name = "NASA Void Helmet"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcomm. This one is in light blue."
+	icon_state = "void-light_blue"
+
+/obj/item/clothing/suit/space/voidsuit/ltblue
+	name = "NASA Void Suit"
+	icon_state = "void-light_blue"

```

Comment:
First off, please make the var order here consistent.
Secondly, why do you define the same `name` on every single type? They use their parent's name by default.

Reply:
I'"" fix the first one, that was something I didn't notice, and the latter was because I changed the names from having colors to not, and no I didn't connect the dots that they didn't need a new name.